.github/autobump.txt

```diff
@@ -2371,6 +2375,7 @@ vsh
 vte3
 vue-cli
 vultr
+vunnel
```

Comment:
this cannot be autobumped

Reply:
see https://github.com/Homebrew/homebrew-core/pull/168254

Reply:
Is there still an issue with autobump for vunnel? The last 5 updates were with `brew bump-formula-pr`.

Reply:
that is the misleading part of the bump-formula-pr, it does not truly reflect the pr status

Reply:
ie, bump-formula-pr can be done with args to ignore some deps cc @branchvincent

Reply:
Should be good to autobump, https://github.com/RedHatProductSecurity/cvss/issues/55 was fixed

```
$ brew update-python-resources vunnel
...
==> Updating resource blocks
```